### PR TITLE
AFRF ,USAF Loadout Fixes

### PR DESCRIPTION
**Pull request by Blazer135**

### CODE REVIEW

**Comment by SonorousMeerkat** [comment]
Travis is drunk. I manually checked that damn file and all of the [ are correctly linked. Can someone work out why the hell it's failing?

**Comment by Godlikehobbit** [comment]
https://github.com/official-antistasi-community/A3-Antistasi/blob/860a93d6f9b38cda419a3abb66d86f6061770c07/A3-Antistasi/Templates/Loadouts/rhs_afrf_marksman.sqf#L42 This is the cause from what I can see

File: A3-Antistasi/Templates/Loadouts/rhs_afrf_machineGunner.sqf [hunk 1]
```diff
@@ -1,36 +1,62 @@
...
+		],
+
+	[//Launcher
+		"rhs_weap_rpg7",								//Weapon
+		"",										//Muzzle
+		"",										//Rail
+		"",										//Sight
+		["rhs_rpg7_PG7VL_mag",1],										//Primary Magazine
+		[],										//Secondary Magazine
+		""										//Bipod
```
Comment (SonorousMeerkat) [inline A3-Antistasi/Templates/Loadouts/rhs_afrf_machineGunner.sqf]: MG has an RPG?

Comment (SonorousMeerkat) [reply A3-Antistasi/Templates/Loadouts/rhs_afrf_machineGunner.sqf]: never mind then, git derped

File: A3-Antistasi/Templates/Loadouts/rhs_afrf_medic.sqf [hunk 2]
```diff
@@ -1,40 +1,75 @@
...
+			["rhs_mag_rdg2_white",1,1],
 			["rhs_mag_rgn",1,1],
-			["rhs_mag_rdg2_white",2,1]]],
-	[
-		"rhs_medic_bag",
-		[] + _medicSupplies
+			["rhs_30Rnd_545x39_7N10_plum_AL",4,30],
+			["rhs_mag_9x18_8_57n181S",2,8]
+		]
+		+ _aceFlashlight
```
Comment (Spoffy) [inline A3-Antistasi/Templates/Loadouts/rhs_afrf_medic.sqf]: I think you might have accidentally removed all the medic's medical gear?

File: A3-Antistasi/Templates/Loadouts/rhs_afrf_medic.sqf [hunk 3]
```diff
@@ -1,40 +1,75 @@
...
+	],
+
+	[//Backpack
```
Comment (Spoffy) [inline A3-Antistasi/Templates/Loadouts/rhs_afrf_medic.sqf]: Removed the medic's backpack too?

**Pull request merged**